src/transforms/objectTransforms.ts

```diff
@@ -61,23 +64,26 @@ export function transformObject(
     description: metadata.description || undefined,
     schema,
     properties: schema.properties
-      ? schema.properties.map(prop => transformProperty(prop))
+      ? schema.properties.map(prop => transformProperty(prop, optionsBag))
       : []
   };
 
   return getAdditionalObjectDetails(objectDetails, uberParents);
 }
 
-export function transformProperty({
-  language,
-  schema,
-  serializedName,
-  required,
-  readOnly,
-  nullable
-}: Property | GroupProperty): PropertyDetails {
+export function transformProperty(
+  {
+    language,
+    schema,
+    serializedName,
+    required,
+    readOnly,
+    nullable
+  }: Property | GroupProperty,
+  optionsBag: OptionsBag
+): PropertyDetails {
   const metadata = getLanguageMetadata(language);
-  const typeDetails = getTypeForSchema(schema);
+  const typeDetails = getTypeForSchema(schema, false, optionsBag.useCoreV2);
```

Comment:
Can you add a comment about what `false` mean?

Reply:
Done